src/bin/jailer.rs

```diff
@@ -7,10 +7,13 @@ extern crate clap;
 extern crate fc_util;
 extern crate jailer;
 
-fn main() -> jailer::Result<()> {
-    jailer::run(
+fn main() {
+    match jailer::run(
```

Comment:
nit: I would have written this clause using `if let Err(e) = jailer::run(..) { eprintln(...,e )}`